### PR TITLE
Blogapp: Feature/forms

### PR DESCRIPTION
**Pull request by mateo951**
- Added form for post creation
- Added form for comment creation
- Added form for like creation
- Add fix for linters

### CODE REVIEW

File: app/views/posts/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  <%= f.hidden_field :comments_counter, value: 0 %><br>
+  <%= f.hidden_field :likes_counter, value: 0 %><br>
+  <div class="flex-row col-flex">
+    <%= f.label :title %>
+    <%= f.text_field :title, size: "50x12", placeholder: "Write your title here..." %>
+    <%= f.label :Text %>
+    <%= f.text_area :text, size: "50x12", placeholder: "Write your post here..." %>
+    <%= f.submit "Submit" %>
+  </div>
+<% end %>
```
Comment (DeliceLydia) [inline app/views/posts/_form.html.erb:12]: - [ ] This form should have its route like `/users/:id/posts/new` it shouldn't be on the user posts page  you can add a button like `add a new post` then it takes you to the new form page

File: config/routes.rb [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
-  get "/users/:id/posts/:id", to: "posts#show"
+  get "/users/:user_id/posts", to: "posts#index"
+  get "/users/:user_id/posts/:post_id", to: "posts#show"
+  get '/users/:id/posts/new', to: 'posts#new', as: 'new_post' # new
+  get '/users/:user_id/posts/:post_id', to: 'comments#new', as: 'new_comment' # new
+  get '/users/:user_id/posts/:post_id/new_like', to: 'likes#new', as: 'new_like' # new
+
+  post '/users/:id/posts/new', to: 'posts#create' # create
+  post '/users/:user_id/posts/:post_id', to: 'comments#create' # create
+  post '/users/:user_id/posts/:post_id/new_like', to: 'likes#create' # create
```
Comment (DeliceLydia) [inline config/routes.rb]: - [ ] Most of the routes are not working check where the issue might be you are supposed to have a route for creating a new post `/users/:id/posts/new` check if you are not making a mistake in your routes
- [ ] A route for creating a new comment and for displaying comments for a specific post are not working `/users/:user_id/posts/:post_id/                                                   comments` `/users/:user_id/posts/:post_id(.:format)                                                          comments/new`

File: app/views/comments/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+      <%= f.text_area :text, size: "50x12",
+      placeholder: "Write your post here...",
+      value: ""
+        %>
+    </p>
+    <p>
+      <%= f.submit %>
+    </p>
+  </div>
+<% end %>
```
Comment (DeliceLydia) [inline app/views/comments/_form.html.erb:16]: - [ ] Comment form should have its route as I mentioned below `/users/:user_id/posts/:post_id(.:format)                                                          comments/new` you can add a button like `add a comment` maybe that lead you to the comment form

File: config/routes.rb (hunk 2 above)
Comment (DeliceLydia) [reply config/routes.rb]: - [ ] Make sure that like has a working route currently `/users/:user_id/posts/:post_id/new_like(.:format) ` doesn't work

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @mateo951, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Major highlights for your project:
- Kudos to the good README
- Followed git workflow
- No linter errors


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: app/views/posts/_form.html.erb (hunk 1 above)
Comment (mateo951) [reply app/views/posts/_form.html.erb:12]: Addressed ✔

File: config/routes.rb (hunk 2 above)
Comment (mateo951) [reply config/routes.rb]: Addressed ✔

File: app/views/comments/_form.html.erb (hunk 3 above)
Comment (mateo951) [reply app/views/comments/_form.html.erb:16]: Addressed ✔

**Review by enaburekhan — APPROVED** [review]
Hi mateo951, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Code review by **Eric Enaburekhan**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ahmetbozaci — COMMENTED** [review]
# STATUS: APPROVED 🥳 🥳 🥳 


@mateo951  ✋🏼 
#### Your project is great. 🎉 I approve of it. There is nothing else to say other than... it's time to merge it. You can submit this project on your dashboard.

___ 

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
aproved

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Pull request merged**